src/main/webapp/app/entities/lecture/lecture.component.html

```diff
@@ -56,14 +56,14 @@ <h2 id="page-heading">
                                     <span class="d-none d-md-inline" jhiTranslate="entity.action.edit">Edit</span>
                                 </button>
                                 <button
-                                    type="submit"
-                                    [routerLink]="['/', { outlets: { popup: 'lecture/' + lecture.id + '/delete' } }]"
-                                    replaceUrl="true"
-                                    queryParamsHandling="merge"
-                                    class="btn btn-danger btn-sm"
+                                    *jhiHasAnyAuthority="['ROLE_ADMIN, ROLE_INSTRUCTOR']"
```

Comment:
Just tested it and it does not work, even if I am instructor in the course and admin. Any idea how we can make this work?